examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -634,7 +636,7 @@ extern "C" void app_main()
 
     // Init ZCL Data Model and CHIP App Server
     AppCallbacks callbacks;
-    InitServer(&callbacks);
```

Comment:
Why does InitServer not require callbacks anymore? Maybe I missed the new place callbacks gets set?